src/goto-symex/symex_builtin_functions.cpp

```diff
@@ -234,8 +229,7 @@ void goto_symext::symex_gcc_builtin_va_arg_next(
   const exprt &lhs,
   const side_effect_exprt &code)
 {
-  if(code.operands().size()!=1)
-    throw "va_arg_next expected to have one operand";
+  PRECONDITION(code.operands().size() != 1);
```

Comment:
Should this be "=="?

Reply:
Yes, you are right, I don't know why this wasn't changed.